### DISCUSSION TITLE
design mismatch

### DISCUSSION THREAD

**Discussion by haaxema** [HASwitchPlate/openHASP#281] [Q&A] [answered]
Hi,

I have one question/remark.

My switches/buttons/sliders do not have the same styling as the ones in the manual.

For example my switches look like:
![swich_live](https://user-images.githubusercontent.com/26124871/149347843-5f03e253-0735-4456-9724-ac12e41dbab9.JPG)

The one in the manual looks like:
![swich_manual](https://user-images.githubusercontent.com/26124871/149348005-f24446e1-820a-4ad4-9564-dc6bb3e8f800.JPG)

Am I doing something wrong?

Martin

**Comment by nagyrobi** [comment, marked as answer]
Go to Plate's web interface > HASP Design > UI Theme and select a different one. Save settings and press Reboot button.

**Reply by haaxema** [reply]
Thanx, completely missed that option......
